src/modules/modals/TransferTable.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+	<NcModal v-if="showModal"
+		size="normal"
+		@close="actionCancel">
+		<div class="modal__content" data-cy="transferTableModal">
+			<div class="row">
+				<div class="col-4">
+					<h2>{{ t('tables', 'Transfer table') }}</h2>
+				</div>
```

Comment:
Do we have to add new text like this to the l10n folder?

Reply:
No need to, once merged, there is a nightly automation job that will push the newly added strings for translation to transifex which is what we use for translators. Then once the text is translated it will be pushed back to the repo automatically.

Full docs for that are in https://docs.nextcloud.com/server/latest/developer_manual/basics/front-end/l10n.html